### ISSUE TITLE
建议把 WindEdit 集成进来

### ISSUE THREAD

**Issue by Jonnyan404** [kingToolbox/WindTerm#135]
我觉得两者的结合,非常方便!
另外:请问如何设置多语言?

**Comment by kingToolbox** [comment]
Yes, these two are a whole at the time of development, so you can download [WindEdit_1.0.0_Prerelease_1](https://github.com/kingToolbox/WindEdit/releases/tag/1.0-prerelease), I believe that is what you want. :smile: 

Multilingual User Interface is still under development and will be available in the next WindTerm_2.2.0 version. In addition, Chinese will also be available at that time. The release time is expected to be in mid-to-late August. Please stay tuned, thank you.

**Comment by Jonnyan404** [comment]
OK, thank you very much! Looking forward to the new version.
